playbooks/bond.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: all
+  become: True
+  tasks:
+    - nmstate_apply:
+        state:
+          interfaces:
+            - name: bond99
+              type: bond
+              state: up
+              ipv4:
+                address:
+                  - ip: 10.10.10.10
```

Comment:
Please consider using RFC 5737 IP addresses here (also something that should be updated in the nmstate.io documentation), see https://tools.ietf.org/html/rfc5737

Reply:
updated

Reply:
I start a new PR about updating nmstate.io documentation , here is the link, https://github.com/nmstate/nmstate.github.io/pull/48